### DISCUSSION TITLE
synchronization project

### DISCUSSION THREAD

**Discussion by ceciliaberon** [opengisch/QField#2375] [Q&A]
hello to the community.
I wrote because I have a question.
I have a project which I work with colleagues who collected information in the field and when I wanted to synchronize the projects to obtain the final result, I observed that when I imported it, it only brought me project A and when I wanted to import project B it did not synchronize them, I only brought them separately.
I do not know if it happened to someone and can give me an idea.
I followed the step by step according to the Qfield document and testing in Qgis.
Clarifications: I wanted to synchronize according to the original project that people were given to collect information.

**Comment by suricactus** [comment]
Can you specify what are the QField, QGIS and QFieldSync versions? A sample subset of the project can help too or at least what type of data storage you use (GeoPackage, PostGIS?)

**Comment by suricactus** [comment]
@ceciliaberon Marking as solved due to inactivity.

**Comment by ceciliaberon** [comment]
Hello, sorry I hay not sean the message, I mandes to solve it, it was an
operational problem on me part.
